core/server/master/src/main/java/alluxio/web/WebInterfaceGeneralServlet.java

```diff
@@ -208,7 +208,7 @@ private void populateValues(HttpServletRequest request) {
     }
 
     String ufsRoot = Configuration.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS);
-    UnderFileSystem ufs = UnderFileSystem.Factory.create(ufsRoot);
+    UnderFileSystem ufs = UnderFileSystem.Factory.createForRoot();
```

Comment:
we should also surround this with try-catch to avoid propagating an exception to the UI